src/pages/profile/Profile.scss

```diff
@@ -0,0 +1,71 @@
+@import '../../index.scss';
+
+.profileContainer {
+    @include mainScreenContainer;
+
+    @include tableScreen {
+        @include tableScreenMainContainer;
+    }
+
+    .profileWrap {
+        @include inputBox;
+        height: fit-content;
+        max-height: 400px;
+
+        @include tableScreen {
+            @include tableScreenWrap;
+            height: fit-content;
+        }
+
+        .title {
+            @include mainScreenTitles;
+
+            @include tableScreen {
+                @include tableScreenTitle;
+            }
+        }
+
+
+        display: flex;
+        flex-direction: column;
+        gap: 5px;
+
+        @include tableScreen {
+            width: 90%;
+        }
+
+        .profileInput {
+            @include initial-inputs;
+        }
+
+        #registerSubmit {
+            @include mainButtonForSubmission;
+            // margin-top: -5px;
+            // margin-bottom: -50px;
+        }
+
+        #avatarInput {
+            display: flex;
+            align-items: center;
+            // gap: 5%;
+            font-size: 1em;
+            color: $primary;
+            width: 20%;
+
+            @include tableScreen {
+                justify-content: center;
+                width: 90%;
+            }
+        }
+
+        #avatarImage {
+            width: 100%;
+        }
+
+        .formError {
+            padding: 0 15px;
+            font-size: 0.85em;
+            color: rgb(187, 1, 1);
+        }
+    }
+}
```

Comment:
Overall please comment the function's purpose. Comments like 'call submit are not descriptive'. Regroup the code so that useStates are together, functions in the order of use, pay attention to async operations. No hints regarding what's going on on the screen like at least what forms are for? What is the current data? Functions are only updating the 'user' collection, what if there are 'userChats' for the user? Validation to be corrected.